### DISCUSSION TITLE
Wondering if Walc Stores contacts

### DISCUSSION THREAD

**Discussion by WillemsenGithub** [cstayyab/WALC#245] [General]
Hi, 
Long story short: lost Whatsapp Android backup, reinstalled Whatsapp, all blank. Of course, Walc also showed blanks when logging back in 

Only hope on recovery of contacts/messages is a recent backup of Walc in Linux (User) folders. 
Would I get lucky ?: I only need to retrieve a contact that got lost in the process.
Where is that data stored, if at all on a Linux Debian Machine ?

Cheers,

**Comment by daffamumtaz2361** [comment]
This app is just a wrapper to the web client. So unless you know how to decrypt the data from browser's storage and import them again somehow, I'm afraid it's not possible.
